### PR TITLE
[doc] Update benchmark command in w4a16.md

### PR DESCRIPTION
**Pull request by del-zhenwu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/w4a16.md [hunk 1]
```diff
@@ -62,8 +62,8 @@ python3 -m lmdeploy.serve.turbomind ./workspace --server_name {ip_addr} ----serv
 
 ```shell
```
Comment (AllentDan) [inline docs/zh_cn/w4a16.md:63]: 可以加下第三方库安装不
```shell
pip install nvidia-ml-py
```

Comment (del-zhenwu) [reply docs/zh_cn/w4a16.md:63]: 好

**Review by AllentDan — APPROVED** [review]
LGTM

**Comment by lvhan028** [comment]
May resolve linting error

File: docs/en/w4a16.md [hunk 2]
```diff
@@ -62,10 +62,14 @@ Memory (GB) comparison results between 4-bit and 16-bit model with context size
...
 
+```
+pip install nvidia-ml-py
+```
+
 ```shell
-python benchmark/profile_generation.py \
-  ./workspace \
-  --concurrency 1 --input_seqlen 1 --output_seqlen 512
+python profile_generation.py \
```
Comment (lvhan028) [inline docs/en/w4a16.md:70]: Is it wrong with `python benchmark/profile_generation.py`?
If changing it to `python profile_generation.py`, we'd better add `cd benchmark`

Comment (del-zhenwu) [reply docs/en/w4a16.md:70]: > Is it wrong with `python benchmark/profile_generation.py`? If changing it to `python profile_generation.py`, we'd better add `cd benchmark`

It works with `benchmark/profile_generation`, I will update it

File: docs/en/w4a16.md [hunk 3]
```diff
@@ -62,10 +62,14 @@ Memory (GB) comparison results between 4-bit and 16-bit model with context size
...
 
+```
+pip install nvidia-ml-py
+```
+
 ```shell
 python benchmark/profile_generation.py \
-  ./workspace \
-  --concurrency 1 --input_seqlen 1 --output_seqlen 512
+ --model-path /path/to/your/model \
```
Comment (lvhan028) [inline docs/en/w4a16.md:71]: I suggest keeping `./workspace` since we've mentioned `./workspace` a lot. 
Users might be confused by `/path/to/your/model`

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**